docs/reference/quotas/openstack.md

```diff
@@ -0,0 +1,48 @@
+---
+description: Resource limits (quotas) for OpenStack resources
+---
+# OpenStack Quotas
+
+Most of the OpenStack resources you create in {{brand}} are subject to quotas.
+Once you hit a quota limit, the creation of new resources of that type will fail, until you either reduce your resource utilization, or your quota is raised.
```

Comment:
```suggestion
Once you hit a quota limit, the creation of new resources of that type will fail, until you either reduce your resource utilization or your quota is raised.
```

Reply:
I guess this is an "Oxford comma" question, but I agree that in this case dropping the comma does improve the reading flow. :)